CHANGELOG.md

```diff
@@ -1,12 +1,222 @@
 # Changelog
 
-## Unreleased
```

Comment:
Should this remain? It will be used for the next release, right?

Reply:
Yes I'll restore